PostmanCollections/Environment/Mojaloop-SIT_environment.json

```diff
@@ -932,9 +932,33 @@
 			"value": "application/vnd.interoperability.transfers+json;version=1.0",
 			"type": "default",
 			"enabled": true
+		},
+		{
+			"key": "payerMSISDN",
+			"value": "449999999",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "payeeMSISDN",
+			"value": "27713803912",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "txnId",
+			"value": "cf1f9731-d248-4cc9-be4c-7937f2ed6c18",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "PAYER_CALLBACK_HOST",
+			"value": "https://webhook.site/724c4049-eced-43d3-9982-7e9ea5bd1d6e",
```

Comment:
can we make sure that this site will be always up ?
is there an alternative? other wise please include documentation about how to set this up. you can add this collection to a directory and add a readme file

Reply:
@danishjamal104 SIT payer callback host should be ML connector ingress.

Reply:
done